Code/GraphMol/ForceFieldHelpers/MMFF/AtomTyper.cpp

```diff
@@ -3651,14 +3651,14 @@ bool MMFFMolProperties::getMMFFVdWParams(const unsigned int idx1,
     const MMFFVdW *mmffVdWParamsIAtom = (*mmffVdW)(iAtomType);
     const MMFFVdW *mmffVdWParamsJAtom = (*mmffVdW)(jAtomType);
     if (mmffVdWParamsIAtom && mmffVdWParamsJAtom) {
-      mmffVdWParams.R_ij_starUnscaled = Utils::calcUnscaledVdWMinimum(
+      mmffVdWParams.R_ij_starUnscaled = MMFF::Utils::calcUnscaledVdWMinimum(
```

Comment:
RDKit::Utils is now in the namespace for localeswitcer... We could change it to something else.